Open_Source_Events/NWOC/README.md

```diff
@@ -0,0 +1,13 @@
+<h1 align="center"> NJACK Winter of Code 2020</h1>
+
+<h2> About NWOC'20 </h2>
+
+NJACK Winter of Code is an initiative by NJACK, IIT Patna, targeted at students who have never participated in Free or Open Source Software (FOSS) development before,
+as well as at those who have already become an expert in Open Source Development and are currently contributing to projects of any domain, say Web, Mobile, Machine Learning,
+Blockchain, IoT etc. It also aims to prepare them for bigger programs and opportunities like Google Summer of Code, Outreachy etc.
+
+<p align="center">
+  <a href="https://njackwinterofcode.github.io/">
+    <img src="https://github.com/Vishal-raj-1/Rotten-Scripts/blob/winter-of-code/Open_Source_Events/NWOC/nwoc.jpg" alt="Logo" />
```

Comment:
Use Imgur Links or relative File Uploads.

Reply:
> Hello @Vishal-raj-1
> 
> Thanks for raising the PR. Let me address the later point first. You will get the points for both the programs since one PR is solving two Issues at the same point. I will ensure that points are shared aptly.
> 
> Secondly, I see that there are so many commits made just for adding two files. Can you try pushing your commits into a single atomic commit?
> 
> You can do that by soft resetting your previous commits. Let me explain how:
> 
> So, lets say: there were 5 commits in total:
> 
> `git reset --soft HEAD~5` will soft reset the last 5 commits.
> 
> Then, you can make a single commit by doing the standard `git add` and `git commit`.
> 
> And after that, `git push -f origin HEAD` will force push the single commit. I will review the PR right now after which you can make all the changes and push the changes to one commit.

Ops Sorry just Check that I was planning to my winter-of-code branch to my master branch then to this repo. But by mistake I have send the PR Directly. Now let me squash for that purpose .